npm install on a fresh dev machine fails

npm install on a fresh dev machine fails

## Expected Behavior
1. Clone InversifyJS master repo. 
2. run npm install
3. no errors displayed in console

## Current Behavior
at step #3 I get

 [15:17:11] Starting 'build-source'...
src\interfaces\bindings\provider.d.ts(4,9): error TS2304: Cannot find name 'Prom
ise'.


## Possible Solution
TypeScript definition, typings package, is missing.
.travis.yml file suggest that typings package is required. 

solution: add the command to the preinstall section of package.json

## Context
Trying to setup th edev. environment for InversifyJS contribution.

## Your Environment

* Version used:
* node - v4.4.0, npm - 3.8.1
* Win 8 x64

--- a/package.json
+++ b/package.json
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "test": "gulp",
+    "preinstall":"npm install -g typings && typings install",
     "prepublish": "gulp build && gulp test && gulp dist"
   },
   "repository": {

